akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardRegion.scala

```diff
@@ -179,7 +175,21 @@ object ShardRegion {
   /**
    * Reply to `GetCurrentRegions`
    */
-  @SerialVersionUID(1L) final case class CurrentRegions(regions: Set[Address]) {
+  @SerialVersionUID(2L) final case class CurrentRegions(regionActorPaths: Set[ActorPath]) {
```

Comment:
add a query for this,```Query("{regex}")```then get the ``CurrentRegions``` to ```RegionsResponse(regionActorPaths: Set[ActorPath],queryHint:Any)```?sorry I don't familiar with sharding ,but I think a query for a dedicated path/sub-path would be  great.